org.gitfourchette.gitfourchette.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.gitfourchette.gitfourchette
+runtime: org.kde.Platform
+runtime-version: '6.7'  # PyQt.BaseApp doesn't support 6.8 yet (https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp#branch-comparison)
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.7'
+cleanup-commands:
+ - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+
+command: gitfourchette
+
+finish-args:
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  # Talk to git remotes
+  - --share=network
+  # Discover ".git" dir associated with files dropped by the user
+  - --filesystem=host:rw
+  # Open directories in system file manager via DBus
+  - --talk-name=org.freedesktop.FileManager1
+  # Start external merge tool via flatpak-spawn
+  - --talk-name=org.freedesktop.Flatpak
+  # Don't mix .pyc files with host environment
+  - --env=PYTHONPYCACHEPREFIX=
```

Comment:
```suggestion
```

I doubt.
This is why there is sandboxing.

Reply:
If we remove this line, and the user has set a custom `PYTHONCACHEPREFIX`, the Flatpak's Python will read/write `.pyc` files in that directory.

I noticed this on my system before adding the line to the manifest. This occurs because the app needs `filesystem=host` (for its git repo discovery feature).

Clearing `PYTHONCACHEPREFIX` forces Python to use the `.pyc` files that ship within the Flatpak (produced during the build phase).